modules/ROOT/partials/events/exportword-events.adoc

```diff
@@ -0,0 +1,7 @@
+The following event are provided by the xref:exportword.adoc[Export to Word plugin].
+
+[cols="1,1,2",options="header"]
+|===
+|Name |Data |Description
+|ExportToWord |N/A |Fired when the editor content is about to be exported as a `docx` file.
+|===
```

Comment:
We don't have any events in this plugin so this section can be removed

Reply:
Removed in lastest commit